buildifier/buildifier.go

```diff
@@ -113,6 +115,14 @@ func main() {
 		os.Exit(2)
 	}
 
+	if *tablesPath != "" {
+		err := tables.ParseAndUpdateJsonDefinitions(*tablesPath)
```

Comment:
The if below, not above.
if *tablesPath != "" {
  if err := tables.ParseAndUpdateJsonDefinitions(*tablesPath); err != nil {
    ...
  }
}